recipes/elfx86exts/build.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+set -e
+cargo install --bin elfx86exts --root $PREFIX
+rm -f $PREFIX/.crates.toml
```

Comment:
What are the side effects to removing this?

Reply:
None, I think? As far as I can tell it's just a stray file that `cargo` creates that doesn't serve any purpose in our context.